scripts/prepare-machine.ps1

```diff
@@ -125,6 +128,10 @@ if ($ForTest) {
     #$InstallCodeCoverage = $true # Ideally we'd enable this by default, but it
                                   # hangs sometimes, so we only want to install
                                   # for jobs that absoultely need it.
+
+    if ($InstallXdpDriver) {
+        $InstallXdpSdk = $true
+    }
```

Comment:
Put this outside of `ForTest`. There's no reason to tie this only to that.

Reply:
Sounds good